init.db/mysql/charity_data.sql

```diff
@@ -0,0 +1,56 @@
+INSERT INTO charity_donation 
+    (financialYear, date, receipt, amount, institution, address, pan, receipts, remarks)
+VALUES 
+    ('2023-2024', '2023-10-11', 'R123456', 1000.00, 'American Charity Foundation', '123 Main St, City, State, 12345', 'ABC123', 'A123, B456', 'General Donation - Helping those in need'),
```

Comment:
The Receipts column in the right hand side was a boolean flag indicating whether or not the receipts received from the beneficiary for the donations. It should have been  a boolean flag or a CHAR(1) to take Y/N - not even CHAR(3) to take Yes/No.

Reply:
Completed. Pls check the latest source code.